journalbeat/plan.sh

```diff
@@ -0,0 +1,52 @@
+pkg_svc_user="root"
+pkg_name="journalbeat"
+pkg_description="Journalbeat is a log shipper from systemd/journald to Logstash/Elasticsearch"
+pkg_origin="core"
+pkg_version="5.5.2"
+pkg_maintainer="smartB Engineering <dev@smartb.eu>"
+pkg_license=('Apache 2')
+pkg_upstream_url="https://github.com/mheese/journalbeat"
+pkg_source="https://github.com/mheese/$pkg_name/archive/v$pkg_version.tar.gz"
+pkg_shasum="3cc431a199226b14e318610ee5cfd08d2f3109dc9850591f73a28219c56753c9"
+pkg_deps=(
+  core/lz4
+  core/systemd
```

Comment:
Just a question - but why is systemd required here?

Reply:
IIRC we need `journalctl`, etc. for the Go binary that is the daemon to work. It's been ages since I looked at this though, so I may not be remembering correctly.

Reply:
See: https://github.com/mheese/journalbeat/blob/master/README.md#cgo